tracks/views.py

```diff
@@ -21,10 +21,12 @@ def get(self, request, genome_id):
                 category_obj = Category.objects.get(id=track.category_id)
                 categories[track.category_id] = CategorySerializer(category_obj).data
                 categories[track.category_id]["track_list"] = []
-            categories[track.category_id]["track_list"].append(ReadTrackSerializer(track).data)
-        return Response([categories[category_id] for category_id in categories], status=status.HTTP_200_OK)
+            categories[track.category_id]["track_list"].append(CategoryTrackSerializer(track).data)
+        return Response({"track_categories": [categories[category_id] for category_id in categories]}, status=status.HTTP_200_OK)
```

Comment:
<img width="343" alt="image" src="https://github.com/Ensembl/ensembl-web-track-api/assets/6834224/64bfe2ed-38ad-4e2d-897c-140e158e47f5">

The order of tracks in right-hand panel should follow the default track order in the genome browser. Compare with beta.

Reply:
Fixed.